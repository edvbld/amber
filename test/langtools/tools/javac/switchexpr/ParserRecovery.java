/**
 * @test /nodynamiccopyright/
 * @compile/fail/ref=ParserRecovery.out -XDrawDiagnostics ParserRecovery.java
 */

public class ParserRecovery {
    void t1(int e) {
         int i = switch (e) { case any; };
    }
    void t2(int e) {
         switch (e) { case any; }
    }
}
