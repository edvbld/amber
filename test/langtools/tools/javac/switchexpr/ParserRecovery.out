ParserRecovery.java:8:39: compiler.err.expected: :
ParserRecovery.java:11:31: compiler.err.expected: :
2 errors
